### ISSUE TITLE
Rebuild for CVE-2019-6111

### ISSUE THREAD

**Issue by nim-nim** [git-for-windows/git#2162]
It would be awfully nice to have a rebuild, to pick up openssh 8.0 and fix CVE-2019-6111 in the git for windows environment

**Comment by PhilipOakley** [comment]
> It would be awfully nice to have a rebuild, to pick up openssh 8.0 and fix CVE-2019-6111 in the git for windows environment

Have a look at the [SDK ](https://gitforwindows.org/#contribute) and the [wiki](https://github.com/git-for-windows/git/wiki) and especially the bits on doing a build. 

It's fairly easy (from what I've seen) to update the 'build-extra' parts to point to the right versions of the support libraries. A PR will then make it available for the next release - the fix may already be there waiting.

https://github.com/git-for-windows/git/wiki/How-to-participate  
https://github.com/git-for-windows/git/wiki/Building-Git  
https://github.com/git-for-windows/git/wiki/Making-an-installer  
https://github.com/git-for-windows/git/wiki/Making-a-portable-Git
etc.

Contributions welcome

**Comment by dscho** [comment]
In addition to actually helping rather than only asking all the time, you could also keep a look-out for the changes to the ReleaseNotes.

You would have found that 5 hours before you opened this ticket, the following commit was added:

https://github.com/git-for-windows/build-extra/commit/ebd9144917a7ddc6f8e3be6066a88cbf88df7d68

And not only that, but also this commit:

https://github.com/git-for-windows/MSYS2-packages/commit/ba8d73cf8431fc7f75bbc6ccd4e07d063bd4668b

And if you would get more involved in this project, you would also learn that the fact that the committer of those two commits is the build agent, it means that there is a new OpenSSH package version in Git for Windows' SDK.

In other words, you would have found out that this request was automagically fulfilled already, even if you did nothing to make it so 😄
